feature/contributors/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/contributors/ContributorsScreen.kt

```diff
@@ -78,6 +79,7 @@ private fun ContributorsScreen(
             null
         }
     val localLayoutDirection = LocalLayoutDirection.current
+    val lifecycleOwner = LocalLifecycleOwner.current
```

Comment:
Good catch! Thanks for finding this issue!
But we can't use this function in commonMain because this function is used by iOS 🙏

Reply:
Oh, My! Ok, I'll fix it